ui/cypress/integration/e2e/lifecyclecheck_spec.js

```diff
@@ -0,0 +1,76 @@
+/// <reference types="cypress" />
+// ***********************************************************
+
+function doGeneralIstioLifecycleChecks(cy) {
+  cy.contains("Lifecycle");
+  cy.contains("Manage Service Mesh Lifecycle");
+  cy.contains("Namespace");
+  cy.contains("Manage Sample Application Lifecycle");
+  cy.contains("Apply Service Mesh Configuration");
+  cy.contains("Validate Service Mesh Configuration");
+  cy.contains("Apply Custom Configuration");
+}
+
+function verifyAllIstioAddons(cy) {
+  cy.contains("Grafana");
+  cy.contains("Jaeger");
+  cy.contains("Kiali");
+  cy.contains("Prometheus");
+  cy.contains("Zipkin");
+}
+
+function doGeneralConsulLifecycleChecks(cy) {
+  cy.contains("Lifecycle");
+  cy.contains("Manage Service Mesh Lifecycle");
+  cy.contains("Namespace");
+  cy.contains("Manage Sample Application Lifecycle");
+  cy.contains("Apply Custom Configuration");
+  cy.get('body').should("not.contain",  "Apply Service Mesh Configuration");
```

Comment:
If [a test case](https://docs.google.com/spreadsheets/d/13Ir4gfaKoAX9r8qYjAFFl_U9ntke4X5ndREY1T7bnVs/edit#gid=0&range=138:138) for Consul Meshery Adapter is defined in Meshery Test Plan (“Click the ‘+’ option in the “Apply Service Mesh Configuration” card and try applying those operations that appear on the service mesh management page.“) why shouldn’t that card be present?